docs/docs/conceptual/gatsby-for-ecommerce.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Gatsby For E-commerce
+---
+
+Businesses selling products online typically need a variety of software to support their experience. At a minimum, their website needs product pages, product catalog navigation, a shopping cart, and checkout. 
+
+Most have additional functionality like customer account creation, discount codes, customer reviews, tax calculation, user tracking via analytics, and content personalization.
```

Comment:
Promotions would be more expansive I think than "discount codes"

Reply:
Also don't forget reverse logistics and loyalty plugins.

Reply:
Ah, yep! Will leave out reverse logistics from since it's not usually part of the core checkout flow, but the rest are great.